api/Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+flask = "*"
```

Comment:
`rq` is a required package which should be added to the pipfile.

I also suspect we should specify the version of flask (and rq) to prevent accidental / unintended version upgrades that might break our code.